test/unit/org/apache/cassandra/cql3/validation/operations/CQLVectorTest.java

```diff
@@ -139,6 +140,95 @@ public void insertNonPK()
         test.run();
     }
 
+    @Test
+    public void cannotInsertWrongNumberOfDimensionsFixedWidth() throws Throwable
```

Comment:
what about change the function name to : wrongNumberOfDimensionsFixedWidth, and the following function names are also the same to remove "cannotInsert", although the function name explains what to do in detail, but I feel a bit long; 
In my mind these tests are all validations for throw exception messages(assertInvalidThrowMessage function everywhere), that is to say the code in the function is saying that wrongNumberxxx will throw an exception.
And wrongNumber and wrongElement is saying this is abnormal error test test(the name is wrongXXX).

Reply:
Yes, the names feel a bit redundant. Changed.